pkg/auth/oauth/registry/registry_test.go

```diff
@@ -340,27 +341,65 @@ func TestAuthenticateTokenOtherGetError(t *testing.T) {
 		t.Errorf("Unexpected user: %v", userInfo)
 	}
 }
+
 func TestAuthenticateTokenExpired(t *testing.T) {
 	fakeOAuthClient := oauthfake.NewSimpleClientset(
 		&oapi.OAuthAccessToken{
-			ObjectMeta: metav1.ObjectMeta{Name: "token", CreationTimestamp: metav1.Time{Time: time.Now().Add(-1 * time.Hour)}},
+			ObjectMeta: metav1.ObjectMeta{Name: "token1", CreationTimestamp: metav1.Time{Time: time.Now().Add(-1 * time.Hour)}}, // expired
```

Comment:
these comments are so far they look silly, what about adding just one explanatory comment on the line before &oapi.OAuthAccessToken{ ?
Like:
// Add expired token with 10 minutes expiration time